frontend/src/app/lightning/nodes-ranking/top-nodes-per-capacity/top-nodes-per-capacity.component.ts

```diff
@@ -14,11 +14,14 @@ import { LightningApiService } from '../../lightning-api.service';
 })
 export class TopNodesPerCapacity implements OnInit {
   @Input() nodes$: Observable<INodesRanking>;
+  @Input() statistics$: Observable<any>;
```

Comment:
Consider creating an interface into the `node-api.interface.ts` similar to `ITopNodesPerChannels` above, instead of using `any`. Basically do a json to ts interface on the server response data.